### PR TITLE
Debian/Ubuntu 14.04 fixes

### PR DESCRIPTION
**Pull request by TJC**
This patchset fixes some showstopper bugs that prevent the role running at all successfully on Ubuntu 14.04 (trusty).

At time of writing, there are bugs later on that cause failures too, but these are, I believe, separate issues. (See issue #21 )

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 - name: Make certain Riak is stopped
```
Comment (christophermancini) [inline tasks/main.yml]: So, @jsmartin is correct in Issue #26 and whenever I add ignore_errors to a task, I shudder a little because it feels like I am just using duct tape to fix a problem. I think if we removed this task, and updated the enabled task to state=restarted, we will get the effect we want of loading the potentially new config and handling for new installs as I just tested (least on Cent7) that if riak is not running, restarted works properly.

Make those edits and I will accept this PR. Thanks man, you have no idea how much I appreciate having an extra set of eyes and help with this repo. :)

Comment (TJC) [inline tasks/main.yml]: I'd actually prefer it if you accepted the current PR, and then I submit another PR for changing the behaviour regarding stop/start stuff. If that's OK?
Keeps these changes separated into logical blocks by what they're changing, rather than one big-bang change.

Comment (mjbrender) [inline tasks/main.yml]: I like your style @TJC - doing so now.
